insights/parsers/smartctl.py

```diff
@@ -1,55 +1,118 @@
+"""
+SMARTctl - command ``/sbin/smartctl -a {device}``
+=================================================
+"""
+
 from insights.core import Parser
 from insights.core.plugins import parser
+from insights.parsers import ParseException
 
 import re
 
 
 @parser('smartctl')
 class SMARTctl(Parser):
     """
-        Parser for output of smartctl -a for each drive in system.
-        This returns a dict with keys:
-         * 'device' - the name of the device after /dev/ - e.g. sda
-         * 'info' - the -i info (vendor, product, etc)
-         * 'health' - overall health assessment (-H)
-         * 'values' - the SMART values (-c) - SMART config on drive firmware
-         * 'attributes' - the SMART attributes (-A) - run time data
-        This function gets called once per detected file, so we don't have to
-        deal with file changes within this process.
+    Parser for output of ``smartctl -a`` for each drive in system.
+
+    This stores the information from the output of `smartctl` in the
+    following properties:
+
+     * ``device`` - the name of the device after /dev/ - e.g. sda
+     * ``info`` - the -i info (vendor, product, etc)
```

Comment:
@PaulWay I believe this should be `information` shouldn't it?  You should also add information about `info` being deprecated in a future version if that change was made in the master branch.